src/Components/CatProfile.tsx

```diff
@@ -84,6 +90,17 @@ function CatProfile() {
 			<h2 className="text-2xl font-bold text-red-950	mb-20">
 				Cat&apos;s Profile
 			</h2>
+			{message && (
+				<div
+					className={`text-2xl font-bold  mb-20
+					${message === 'Updates have been saved' ? 'text-red-950' : ''}
+					${message === 'Updates havent been saved' ? 'text-red-950' : ''}
+					${message === 'Profile Removed' ? 'text-red-950' : ''}
+					${message === 'Failed to delete the profile' ? 'text-red-950' : ''}`}
```

Comment:
All styling should be in components now. See `src/Components/Styling/FormElement.tsx` for an example.

Reply:
Components are now in #17 awaiting merging to main

Reply:
Todo in CL-119